frontend/src/pages/PipelineDetails.tsx

```diff
@@ -131,8 +131,13 @@ class PipelineDetails extends Page<{}, PipelineDetailsState> {
       // Add buttons for creating experiment and deleting pipeline
       actions = actions.concat([
         buttons.newExperiment(() => this.state.pipeline ? this.state.pipeline.id! : ''),
-        buttons.delete(() => this.state.pipeline ? [this.state.pipeline.id!] : [],
-          'pipeline', () => null, true),
+        buttons.delete(
+          () => this.state.pipeline ? [this.state.pipeline.id!] : [],
+          'pipeline',
+          this._deleteCallback.bind(this),
+          true, /* useCurrentResource */
+          false, /* refreshOnComplete */
```

Comment:
If there's a good reason why we don't refresh after delete, it's worth calling it out in a comment here.

Reply:
Done